server/src/main/proto/spine_examples/kanban/board.proto

```diff
@@ -64,15 +80,10 @@ message BoardInit {
     option (entity).kind = PROCESS_MANAGER;
 
     // The ID of the board we initialize.
-    BoardId id = 1 [(required) = true];
-
-    // Columns already placed.
-    repeated ColumnId placed_column = 3;
+    BoardId id = 1;
 
-    // The process manager used to create default columns one by one, "step by step".
-    // Currently, it creates them all at once.
-    reserved 2;
-    reserved "step";
+    // Columns already added.
+    repeated ColumnId added_column = 3;
```

Comment:
Look, if you have chosen to remove the "reserved" fields, you need to adjust the field numbers as well.